nixos/modules/services/computing/slurm/slurm.nix

```diff
@@ -220,12 +276,22 @@ in
         '';
       };
 
-  in mkIf (cfg.enableStools || cfg.client.enable || cfg.server.enable) {
+  in mkIf ( cfg.enableStools ||
+            cfg.client.enable ||
+            cfg.server.enable ||
+            cfg.dbdserver.enable ) {
 
     environment.systemPackages = [ wrappedSlurm ];
 
     services.munge.enable = mkDefault true;
 
+    # use a static uid as default to ensure it is the same on all nodes
+    users.users.slurm = mkIf (cfg.user == defaultUser) {
+      name = defaultUser;
+      group = "slurm";
```

Comment:
you need to explicitly create the group with the reserved gid, as seen in these error messages from the slurm test:
```
node2# warning: user ‘slurm’ has unknown group ‘slurm’
node3# warning: user ‘slurm’ has unknown group ‘slurm’

```

Reply:
Yep, sorry. fixed. The test succeeds again.